stock_move_line_lock_qty_done/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    lock_qty_done = fields.Boolean(
+        default=True,
```

Comment:
But last question, why enabling it by default?

IMHO, that should be done on your project side.

Reply:
My thought is that users install this module because they want to use its features. So, requiring them to change the settings after installing the module feels a bit strange to me.

Reply:
That has been always a debate. But that changes Odoo standard behavior. Which is not always wanted. That's why enabling the feature on deployment is better.